README.md

```diff
@@ -0,0 +1,29 @@
+# High Level Controller
+
+## Overview
+
+The control-high repository consists of the High-Level Controller and the MHIST submodule of the SOZO Robotics Project.
+
+The High-Level Controller (HLC) receives the area to cover from the Swarm Algorithm (tbd, currently mocked), gets updates about the local map and the current position in the map from [SLAM](https://github.com/codeuniversity/slam) and feedback about the (sucessful) leg movement from the [gait controller](https://github.com/codeuniversity/control-gait).
```

Comment:
I would put `(successful)` after `feedback` and add what other feedback the controller could get (`panic` | `leg x blocked` or something?)